nototools/mti_cmap_data.py

```diff
@@ -111,7 +111,7 @@ def cmap_data_from_csv(
   metadata = cmap_data.create_metadata('mti_cmap_data', args)
   script_to_cmaps = get_script_to_cmaps(csvdata)
   if scripts or exclude_scripts:
-    script_list = script_to_cmap.keys()
+    script_list = script_to_cmaps.keys()
```

Comment:
well spotted. I guess nobody has ever run this..

Reply:
I'm not sure anybody has ever run half this stuff.